crates/compute_test_utils/src/bivariate_sumcheck.rs

```diff
@@ -255,3 +261,163 @@ where
 		.map(|j| witness.evaluate_on_hypercube(j).unwrap())
 		.sum()
 }
+
+fn evaluate_composite_at_point<F, M, Composition>(
+	multilinears: &[M],
+	composition: Composition,
+	eval_point: &[F],
+) -> F
+where
+	F: Field,
+	M: MultilinearPoly<F> + Send + Sync,
+	Composition: CompositionPoly<F>,
+{
+	let n_vars = multilinears
+		.first()
+		.map(|multilinear| multilinear.n_vars())
+		.unwrap_or_default();
+	for multilinear in multilinears {
+		assert_eq!(multilinear.n_vars(), n_vars);
+	}
+
+	assert_eq!(eval_point.len(), n_vars);
+
+	let multilinears = multilinears.iter().collect::<Vec<_>>();
+
+	let backend = make_portable_backend();
+
+	let eq_ind = EqIndPartialEval::new(eval_point)
+		.multilinear_extension::<F, _>(&backend)
+		.unwrap();
+
+	let multilinears = multilinears.iter().collect::<Vec<_>>();
+	let witness = MultilinearComposite::new(n_vars, composition, multilinears.clone()).unwrap();
```

Comment:
nit: Use `MultilinearComposite::evaluate` method instead

Reply:
Ah, sorry, you're right. This would actually be a useful method to add: `MultilinearComposite::evaluate_mle(query: impl AsRef<MultilinearQuery<F>>) -> F`.